pipelines/templates/buildpack.yml.erb

```diff
@@ -522,15 +453,34 @@ jobs: ##########################################################################
                 contents="$(jq '.stack = "<%= stack %>"' config.json)"
                 echo -E "${contents}" > config.json
                 echo -e "config.json modified to:\n $(cat config.json)"
-<% end %>
+      <% end %>
         - task: brats-<%= stack %>
           file: buildpacks-ci/tasks/<%= language.to_s == 'php' ? 'run-bp-brats-jammy' : 'run-bp-brats' %>/task.yml
           input_mapping: {cf-space: space}
+          attempts: 5
           params:
             CF_STACK: <%= stack %>
             GINKGO_ATTEMPTS: 4
             GINKGO_NODES: 3
-<% end %>
+          <% end %>
+        - task: integration-test-<%= stack %>
+          file: buildpacks-ci/tasks/run-buildpack-integration-specs/task.yml
+          input_mapping: {cf-space: space}
+          params:
+            CF_STACK: <%= stack %>
+            GINKGO_ATTEMPTS: 4
+            GINKGO_NODES: 3
+            <% if language == 'php' %>
+            COMPOSER_GITHUB_OAUTH_TOKEN: '((composer-github-oauth-token))'
+            <% end %>
+            <% if buildpacks[language]['skip_docker_start'] %>
+            SKIP_DOCKER_START: true
+            <% else %>
+          privileged: true
+            <% end %>
+        ensure:
+          task: delete-cf-space
```

Comment:
It looks like we need the the create-cf-space task so that the input dir to the task exists. https://buildpacks.ci.cf-app.com/teams/main/pipelines/go-buildpack/jobs/specs-edge-integration-develop-cflinuxfs3/builds/108